schemas/com.mbta.rtr.light-rail-vehicle-information.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/mbta/schemas/blob/main/schemas/com.mbta.rtr.light-rail-vehicle-information.json",
+  "description": "Light Rail Vehicle Information",
+  "type": "object",
+  "properties": {
+    "id": {
+      "$ref": "cloudevents.json#/properties/id",
+      "examples": ["123abc"]
+    },
+    "source": {
+      "$ref": "cloudevents.json#/properties/source",
+      "examples": ["mbta.com/rtr-1"]
+    },
+    "specversion": {
+      "const": "1.0",
+      "$ref": "cloudevents.json#/properties/specversion"
+    },
+    "type": {
+      "const": "com.mbta.rtr.light-rail-vehicle-information",
+      "$ref": "cloudevents.json#/properties/type"
+    },
+    "time": {
+      "$ref": "cloudevents.json#/properties/time"
+    },
+    "data": {
+      "description": "The event payload.",
+      "$ref": "#/$defs/datadef"
+    }
+  },
+  "required": ["type", "specversion", "source", "id", "time", "data"],
+  "$defs": {
+    "datadef": {
+      "type": "object",
+      "required": [
+        "directionId",
+        "lastAvi",
+        "vehicleId",
+        "routeId",
+        "consist",
+        "orientation",
+        "nextStopId",
+        "stopStatus",
+        "lat",
+        "lon"
+      ],
+      "properties": {
+        "directionId": {
+          "description": "Direction in which the vehicle is headed. Derived from the direction in trips.txt: https://gtfs.org/documentation/schedule/reference/#tripstxt",
+          "type": "integer"
```

Comment:
This could have `"enum": [0, 1]`, cuz not every integer is a valid direction.

[docs](https://json-schema.org/understanding-json-schema/reference/enum) [example](https://github.com/mbta/schemas/blob/4c580eac6ae205311e5675ec153f2565a9881d38/schemas/com.mbta.ctd.subwayside.train-heartbeat.json#L273)